src/modules/src/Kconfig

```diff
@@ -150,16 +150,6 @@ config MOTORS_REQUIRE_ARMING
         When enabled, system must be armed to be able to take off. Arming can
         be done in several ways, e.g. though cfclient or external transmitter.
 
-config MOTORS_DEFAULT_IDLE_THRUST
```

Comment:
Is this supposed to be removed?

Reply:
Currently, having a parameter both defined in the platform defaults file AND in kconfig is incompatible. 
```
error: "CONFIG_MOTORS_DEFAULT_IDLE_THRUST" redefined [-Werror]
```

Given [our rule of thumb](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/userguides/platform/#persistent-paramters) I thought it was more logical to not have it as a compile time parameter. With the implementation in this PR it still defaults to 0 for platforms that do not have it defined in their platform defaults and the value can always be changed through the parameter framework. 

I think this also covers the case for a custom build with the Bolt where the response to the idle thrust may be unknown, since it defaults to 0 for that platform.

What do you think?

Reply:
I think that makes sense!